markdown-mode.el

```diff
@@ -6099,10 +6106,23 @@ not at a link or the link reference is not defined returns nil."
     (match-string-no-properties 2))
    (t nil)))
 
-(defun markdown-follow-link-at-point ()
-  "Open the current non-wiki link in a browser."
-  (interactive)
-  (if (markdown-link-p) (browse-url (markdown-link-link))
+(defun markdown-link-filename (name)
+  "Remove fragment from file links."
+  (string-match markdown-regex-link-fragment name)
+  (match-string 1 name))
+
+(defun markdown-follow-link (name &optional other)
+  "Follow the link NAME in a browser.
+Open the with `find-file' when OTHER is non-nil."
+  (if other (find-file (markdown-link-filename name))
```

Comment:
Is `ffap` better rather than `find-file` ? If the link is HTTP URL(like `http://github.com/docker/docker/blob/master/README.md`), `C-u M-x markdown-follow-link-at-point` creates terrible file.

And should file open function be customizable ? (For example default value is `find-file` or `ffap` and user sets find-file-other-window or their own function).